### PR TITLE
Update binaries to the latest version

### PR DESCRIPTION
**Pull request by Roming22**
Signed-off-by: Romain Arnaud <rarnaud@redhat.com>

### CODE REVIEW

**Review by gabemontero — APPROVED** [review]
tests are passing 

/lgtm

File: shared/config/dependencies.sh [hunk 1]
```diff
@@ -1,15 +1,15 @@
 #!/usr/bin/env bash
-export ARGOCD_VERSION="v2.4.5"
-export BITWARDEN_VERSION="v2022.10.0"
-export CHECKOV_VERSION="2.2.254"
-export GO_VERSION="1.19.4"
-export GRPC_CLI_VERSION="v1.50.0"
-export HADOLINT_VERSION="v2.10.0"
+export ARGOCD_VERSION="v2.3.13"
```
Comment (ramessesii2) [inline shared/config/dependencies.sh:2]: This seems to be an older version to `ARGOCD_VERSION="v2.4.5"`

Comment (Roming22) [reply shared/config/dependencies.sh]: Good catch. Fixed. I'll work on a patch to the task so it does not happen again.

**Review by ramessesii2 — APPROVED** [review]

**Pull request merged**